src/tests/jira.test.ts

```diff
@@ -0,0 +1,181 @@
+import { it, describe, expect } from "vitest";
+import { AxiosMock } from "./mockClients";
+import JiraClient from "jira-client";
+import { Jira } from "../libs";
+import * as mockResponses from "./mockResponses";
+
+describe("Jira tests", () => {
+  testJiraClientSearchJira();
+  testJiraClientSearchHandlesZeroResults();
+  testJiraLibSearchWithClosedStatuses();
+  testThrowsErrorIfSearchQueryMissingAwsAccountId();
+  testThrowsExceptionInGetAllSecurityHubIssuesInJiraProject();
+  testErrorRemovingWatcherFromJiraIssue();
+  testErrorClosingJiraIssue();
+  testMissingRequiredEnvVar();
+  testErrorCreatingJiraIssue();
+  testCannotFindDoneTransitionForIssue();
+  testTransitionIssue();
+});
```

Comment:
This is an interesting pattern I haven't seen before! Is there a reason you wrapped each test into a function and then called those functions in the describe block? They all seem to run and pass, but I'm curious if there was a reason you didn't nest the `it` statements directly in the `describe` block.

Reply:
There isn't really a reason.  I was trying out different ideas to organize the tests.  I ended up going with this one because I liked that I could easily see the entire describe block on one screen.  Though it did lead to the common issue that naming things is hard.  I'm very open to other ideas, not locked in on this pattern.